independent-publisher-connectors/IBM Watson Assistant/apiProperties.json

```diff
@@ -0,0 +1,76 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "url": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Service URL",
+          "description": "The service URL",
+          "tooltip": "Provide your service URL",
+          "constraints": {
+            "tabIndex": 1,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "environment": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Environment ID",
+          "description": "The Environment ID",
+          "tooltip": "Provide the environment",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "username": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "API Key Name",
+          "description": "The API key name",
+          "tooltip": "Provide the username",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true",
+            "allowedValues": [
+              {
+                "text": "Default",
+                "value": "apikey"
+              }
+            ]
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The service API key",
+          "tooltip": "Provide the API Key",
+          "constraints": {
+            "tabIndex": 4,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0F62FE",
```

Comment:
@lucastitus for all our independent publishers, we ask to set the iconBrandColor to "#da3b01". Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).